### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by rxqbi** [BisheshS/Virtual-Clothes-TryOn#4]
absl-py==1.0.0
astunparse==1.6.3
cachetools==5.0.0
certifi==2021.10.8
charset-normalizer==2.0.12
click==8.0.4
Flask==2.0.3
Flask-Uploads==0.2.1
flatbuffers==2.0
gast==0.5.3
google-auth==2.6.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.44.0
h5py==3.6.0
idna==3.3
imageio==2.16.0
importlib-metadata==4.11.2
itsdangerous==2.1.0
Jinja2==3.0.3
joblib==1.1.0
keras==2.8.0
Keras-Preprocessing==1.1.2
libclang==13.0.0
Markdown==3.3.6
MarkupSafe==2.1.0
networkx==2.6.3
numpy==1.22.2
oauthlib==3.2.0
opencv-python==4.5.5.62
opt-einsum==3.3.0
packaging==21.3
Pillow==9.0.1
protobuf==3.19.4
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyparsing==3.0.7
PyWavelets==1.2.0
requests==2.27.1
requests-oauthlib==1.3.1
rsa==4.8
scikit-image==0.19.2
scikit-learn==1.0.2
scipy==1.8.0
six==1.16.0
sklearn==0.0
tensorboard==2.8.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.8.0
tensorflow-io-gcs-filesystem==0.24.0
termcolor==1.1.0
tf-estimator-nightly==2.8.0.dev2021122109
threadpoolctl==3.1.0
tifffile==2022.2.9
typing-extensions==4.1.1
urllib3==1.26.8
Werkzeug==2.0.3
wrapt==1.13.3
zipp==3.7.0

**Comment by rxqbi** [comment]
in >>>  from werkzeug import secure_filename,FileStorage 
change it to >>> from werkzeug.utils import secure_filename
from werkzeug.datastructures import  FileStorage

**Comment by rxqbi** [comment]
Done
